downloader/src/main/kotlin/Downloader.kt

```diff
@@ -302,7 +302,12 @@ object Downloader {
                     // So first look for a dedicated header in the response, but then also try both redirected and
                     // original URLs to find a name which has a recognized archive type extension.
                     response.headers("Content-disposition").mapNotNullTo(candidateNames) { value ->
-                        value.withoutPrefix("attachment; filename=")
+                        val tokens: List<String> = value.split(";")
```

Comment:
Commit message nit: "filename =" should have no space after "filename" and say "filename="

In total (ready for copy & paste):

```
The "content-disposition" header may have more than one "filename="
pattern, see https://tools.ietf.org/html/rfc6266#section-4.1.
```